docs/contributing/integration-tests.md

```diff
@@ -0,0 +1,16 @@
+# Integration tests
+
+## Testing most popular datasets
+
+Some integration tests operate on the most popular collections for each provider in CMR.
+Those collections are cached as static data in `tests/integration/popular_collections/`
```

Comment:
```suggestion
Those collection IDs are cached as static data in `tests/integration/popular_collections/`
```

I read this like we cached all the data, not just the names of the collections we care about.

Reply:
:100: